clients/cloud/java/README.md

```diff
@@ -107,7 +109,7 @@ The Kafka Streams API reads the same topic from Confluent Cloud and does a state
 
     # Run the Kafka streams application
     $ mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.StreamsExample" \
-      -Dexec.args="$HOME/.ccloud/config test1"
+      -Dexec.args="$HOME/examples/clients/cloud/java/example.config test1"
```

Comment:
Same issue about full path as mentioned in the Groovy example.  Let's simplify to `example.config` if possible